ksqldb-execution/src/test/java/io/confluent/ksql/execution/builder/KsqlQueryBuilderTest.java

```diff
@@ -281,4 +282,60 @@ public void shouldTrackValueSchemasUsed() {
             SerdeFeatures.of(SerdeFeature.WRAP_SINGLES)))
     )));
   }
+
+  @Test
```

Comment:
Changes in this file are related to review comment in https://github.com/confluentinc/ksql/pull/6365